labgrid/environment.py

```diff
@@ -43,6 +43,12 @@ def get_target(self, role: str = 'main') -> Target:
 
         return self.targets[role]
 
+    def get_feature_flags(self):
+        return self.config.get_flags()
```

Comment:
This is broken at this point in the series. Also, 'environment: return flags for all targets' changes this code completely.